browser/src/control/Control.Notebookbar.js

```diff
@@ -451,9 +451,11 @@ L.Control.Notebookbar = L.Control.extend({
 	onRulerChange: function() {
 		if (this.map.uiManager.isRulerVisible()) {
 			$('#showruler').addClass('selected');
+			document.documentElement.style.setProperty("--canvas-container-y", "var(--ruler-height)");
```

Comment:
I think we should prepare some helper for such things:
`app.setCssVariable("name", "value")`
and
`app.setGlobalCssClass("class-name")` - which will put class into root element in the DOM, like we use to later style app depending on app-type, mode, etc

Reply:
understood

Reply:
but it that case we can get away without any new helper I can just apply a new class that would probably be more useful for other things in the long run (so CSS knows when the ruler is on or off just by the class selector

Reply:
yes, but helper will be nice - one unified way to add these "flags", so they are in one place and it is easy to target them then in CSS.
Also easier to find them in the code by searching the function name.
It's just an idea for later.